DMCompiler/DMStandard/Types/Image.dm

```diff
@@ -1,17 +1,17 @@
 ﻿/image
 	parent_type = /datum
 
-	var/alpha
+	var/alpha = 255
```

Comment:
The defaults for `/image` don't come from here, they're whatever the default value in `IconAppearance` is.

I suppose setting it here is desirable for `initial()` though. Maybe leave a comment somewhere clarifying where the actual defaults come from?

Reply:
So I should set these values in `IconAppearance` and `/image`?

Reply:
These values are all already set in IconAppearance, so I just added the comment.